sdk/java-sdk/src/main/java/kalix/javasdk/action/Action.java

```diff
@@ -148,6 +148,13 @@ interface Builder {
        * @param <S> The type of the message that must be returned by this call.
        */
       <S> Effect<S> asyncEffect(CompletionStage<Effect<S>> futureEffect);
+
+      /**
+       * Ignore the current element and proceed with processing the next element if returned for an
+       * element from eventing in. If used as a response to a regular gRPC or HTTP request it is turned
+       * into a NotFound response.
+       */
```

Comment:
```suggestion
       * 
       * Ignore is not allowed to have side effects added with `addSideEffects`
       */
```

and same in the Scala one